observer/bin/spark-observer.js

```diff
@@ -47,7 +43,14 @@ const loop = async (name, fn, interval) => {
     }
     const dt = Date.now() - start
     console.log(`Loop "${name}" took ${dt}ms`)
-    await timers.setTimeout(interval - dt)
+    recordTelemetry(`loop_${name.replaceAll(' ', '_')}`, point => {
```

Comment:
`name` is meant to be a human readable string, not an identifier. I think we should either add a telemetry identifier `{ id: 'transfer-events' }` or use a safe method of converting that doesn't produce failures if we use a name like `"oh:what?!"`, like https://www.npmjs.com/package/slugify